src/loaders/assetLoader.js

```diff
@@ -129,7 +130,7 @@ module.exports = async function assetLoader() {
     this.emitFile(dest, item.content);
   });
 
-  let publicPath = `__webpack_public_path__ + ${JSON.stringify(path.join('/', assets, url))}`;
+  let publicPath = `__webpack_public_path__ + ${JSON.stringify(path.join('/', assets, url).replace(pathSepPattern, '/'))}`;
```

Comment:
That code is used in the react-native runtime, on [/Libraries/Image/AssetSourceResolver.js#L27](https://github.com/facebook/react-native/blob/1fa95ed390d96b4a54702acdf89594af1785a3fd/Libraries/Image/AssetSourceResolver.js#L27), to resolve the asset filename ([/Libraries/Image/resolveAssetSource.js](https://github.com/facebook/react-native/blob/1fa95ed390d96b4a54702acdf89594af1785a3fd/Libraries/Image/resolveAssetSource.js))